### DISCUSSION TITLE
Does Free Disc Space Take into Account Mounted Drives?

### DISCUSSION THREAD

**Discussion by rebeltaz** [bleachbit/bleachbit#1495] [Q&A] [answered]
Under Linux, using BleachBit v4.4.2, if you select Free Disk Space, will that also wipe the free space on mounted drives? If not, is there any way to ensure that it does?

**Comment by az0** [comment, marked as answer]
If you want to clean it every time, do this

1. Mount the removable drive
2. Open BleachBit
3. Open the preferences (hamburger menu)
4. Click the Drives tab
5. Add the folder there
6. Close preferences
7. Clean free space

If you want to do it just once

1. Mount the removable drive
2. Open BleachBit
3. Open the hamburger menu
4. Click shred folders
5. Choose the removable drive

**Reply by rebeltaz** [reply]
Thank you.
